### ISSUE TITLE
gmtime64

### ISSUE THREAD

**Issue by crotwell** [EarthScope/dali2dali#2]
Missing on my ubuntu 22.04 system.

```
scsn@eeyore2023:/data/scsn/src/dali2dali-0.2$ make
make -C libdali 
make[1]: Entering directory '/data/scsn/src/dali2dali-0.2/libdali'
cc   -c timeutils.c -o timeutils.o
timeutils.c:29:10: fatal error: gmtime64.h: No such file or directory
   29 | #include "gmtime64.h"
      |          ^~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:95: timeutils.o] Error 1
make[1]: Leaving directory '/data/scsn/src/dali2dali-0.2/libdali'
make: *** [Makefile:8: libdali] Error 2
```

**Comment by crotwell** [comment]
Compiles fine if I replace libdali with the 1.8.1 release. Any reason that is not safe to do?

**Comment by chad-earthscope** [comment]
Whoops!  New files from the updated library were not added to the repository.

Fixed in v0.3: https://github.com/EarthScope/dali2dali/releases/
